specification/graphservicesprod/resource-manager/readme.md

```diff
@@ -49,7 +58,7 @@ input-file:
 This section describes what SDK should be generated by the automatic system.
 This is not used by Autorest itself.
 
-```yaml $(swagger-to-sdk)
+``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-go
```

Comment:
Could we add `azure-sdk-for-net-track2` into this list to enable the generation checks for dotnet as well?

Reply:
Added